src/mscorlib/src/System/TimeZoneInfo.cs

```diff
@@ -282,12 +284,23 @@ private AdjustmentRule GetAdjustmentRuleForAmbiguousOffsets(DateTime adjustedTim
         /// If the specified rule is the first AdjustmentRule, or it isn't in _adjustmentRules,
         /// then the specified rule is returned.
         /// </summary>
-        private AdjustmentRule GetPreviousAdjustmentRule(AdjustmentRule rule)
+        private AdjustmentRule GetPreviousAdjustmentRule(AdjustmentRule rule, int? ruleIndex)
```

Comment:
Do you mean just taking `int ruleIndex`?  I don't think there would be any callers of it, since the method that returns the rule can return `null` for the rule, and it returns `null` for the index as well.